build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'com.github.hmcts'
-version '1.18.0'
+version '1.18.1'
```

Comment:
Since this is a breaking change (singular file uploads moving to multiple uploads). Would it make sense to make this a MAJOR bump instead of PATCH?